pkg/service/test/test.go

```diff
@@ -509,10 +512,12 @@ func (t *tester) SimulateRequest(cfg *SimulateRequestConfig) {
 		testPass, testResult := t.testHttp(*cfg.Tc, resp, cfg.NoiseConfig, cfg.IgnoreOrdering)
 
 		if !testPass {
+			t.logger.Debug("", zap.Any("matched mocks", cfg.LoadedHooks.GetMatchedMocks()))
```

Comment:
incase if debug flag is enabled, the logs are just too much and we will loose even the test run summary and other context.

It's better to use info log as enabling debug flag 
<img width="1136" alt="image" src="https://github.com/keploy/keploy/assets/60891544/df2dcaf6-6f4b-4358-ac46-d26a6f498c10">

Reply:
done!